otx/core/data/caching/storage_cache.py

```diff
@@ -0,0 +1,96 @@
+"""A thin wrapper for storage cache using datumaro arrow format exporter."""
+# copyright (c) 2023 intel corporation
+# spdx-license-identifier: apache-2.0
+#
+
+import hashlib
+import os
+import stat
+from typing import List, Optional
+
+from datumaro.components.dataset import Dataset as DatumDataset
+
+from otx.core.file import OTX_CACHE
+
+DATASET_CACHE = os.path.join(OTX_CACHE, "dataset")
+
+
+def arrow_cache_helper(
+    dataset: DatumDataset,
+    scheme: str,
+    num_workers: int = 0,
+    cache_dir: str = DATASET_CACHE,
+    force: bool = False,
+) -> List[str]:
+    """A helper for dumping Datumaro arrow format."""
```

Comment:
please provide the descriptions for arguments. If `force` is set to True, `cache_hit` is going to be `False`. Is this under your intention?

Reply:
Added description in https://github.com/openvinotoolkit/training_extensions/pull/2009/commits/094005491b86f5d109367b1ce337b7cec9b52ecd and make `force` bold.